### ISSUE TITLE
Feature request: reordering accented letters

### ISSUE THREAD

**Issue by avariado** [klausw/hackerskeyboard#511]
Would it be possible to customize accented letters order? I'm from Portugal and it takes too long for me to pick letters like ó ô õ ê é í ú which are very common, as they are too far. It would be awesome if I could rearrange them myself to my own needs.

**Comment by yassine987** [comment]
unfortunately, you can't change accented letters order as a user, but if you are a familiar with Android studio or eclipse I can guide you to rearrange them by yourself, let's take the **e** key in the french keyboard as an example:
go to the french (fr) keyboard XML in your resource folder `java/res/xml-fr/kbd_qwerty.xml` and look for `android:keyLabel="e"` [Line40](https://github.com/klausw/hackerskeyboard/blob/master/java/res/xml-fr/kbd_qwerty.xml#L40), the accented letters are in the `android:popupCharacters` if you are on the right path the line must be looks like this `android:popupCharacters="@string/alternates_for_e"` [Line42](https://github.com/klausw/hackerskeyboard/blob/master/java/res/xml-fr/kbd_qwerty.xml#L42), now we have to change the order from the **alternates_for_e** , you can find it in `java/res/values-fr/donottranslate-altchars.xml` [Line22](https://github.com/klausw/hackerskeyboard/blob/master/java/res/values-fr/donottranslate-altchars.xml#L22), now the only thing you have to do is changing the letters between the tag (`3èéêë€` in our case) `<string name="alternates_for_e">3èéêë€</string>`

hope this is helpful for your issue.

**Comment by avariado** [comment]
yassine987 WOW... that's awesome. I'm kind of newbie on those issues, but I'm aware of those softwares and even tried editing apk before with APK Multi-Tool, so I understand what you're saying and definitely will try it anytime soon. Thank you so much for taking your time to share with me those amazing tips, I really appreciated that. :)

**Comment by avariado** [comment]
YES! I knew I would get there, thanks to your help I "hacked" Hacker's Keyboard, haha :D 
I used the tool I knew (APK-Multi-Tool-WIP-2.0), perhaps not the best method but it did the job. Made the changes on \res\values-pt\strings.xml, recompiled, and then I was afraid not to be able to sign the apk but the script option nº 20 worked fine. 
So this is perfect, now I can write faster than ever on this great keyboard, **thank you so much, yassine987!**

**Comment by yassine987** [comment]
Nice to hear that ^_^
